app/src/main/java/org/simple/clinic/selectcountry/SelectCountryUpdate.kt

```diff
@@ -24,11 +23,13 @@ class SelectCountryUpdate : Update<SelectCountryModel, SelectCountryEvent, Selec
   }
 
   private fun countrySaved(model: SelectCountryModel): Next<SelectCountryModel, SelectCountryEffect> {
-    return if (model.hasSelectedACountry() && !model.hasMoreThanOneDeployment) {
+    val effect = if (model.hasSelectedACountry() && !model.hasMoreThanOneDeployment) {
```

Comment:
No particular reason tbh, to I was just being extra careful I guess. It's expected to have a selected state at this point, so there is no need to have this, and since the model will be restored during process deaths properly. There is no need to have it. I can remove this.